docs/examples/balltree-kdtree-performance.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>tabulate and make final remarks on the observations. </p><p>It'd be good to have 3 types of meshes small/med/large</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I've provided a 30km and 3.75km mesh, but also have 15km and 7.5km avaliable if we want to use those.